aws/data_source_aws_instance_test.go

```diff
@@ -429,6 +444,42 @@ data "aws_instance" "foo" {
 }
 `
 
+const testAccInstanceDataSourceConfig_PlacementGroup = `
+resource "aws_vpc" "foo" {
+  cidr_block = "10.1.0.0/16"
+  tags {
+    Name = "testAccInstanceDataSourceConfig_PlacementGroup"
+  }
+}
+
+resource "aws_subnet" "foo" {
+  cidr_block = "10.1.1.0/24"
+  vpc_id = "${aws_vpc.foo.id}"
+}
+
+resource "aws_placement_group" "foo" {
+  name = "testAccInstanceDataSourceConfig_PlacementGroup"
```

Comment:
We will need to add some name randomization here, so that we avoid conflicts when running tests